R/get_diff_genes.R

```diff
@@ -34,9 +39,9 @@ get_diff_genes<- function(mrobj, genome, threshold = 0, qvalue = 1){
 		# change the treatment list to a c(0:length(treatment_list))
 		mrobj <- reorganize(mrobj, treatment = 0 : (length(treatment_list) - 1))
 		# unite the mrobj
-		meth <- unite(mrobj, destrand = TRUE, mc.cores = 8)
```

Comment:
Setting the number of cores to use to 8 may lead to issue when running the package on a computer with less than 8 cores.